src/node_errors.h

```diff
@@ -80,6 +80,7 @@ void OnFatalError(const char* location, const char* message);
   V(ERR_VM_MODULE_LINK_FAILURE, Error)                                         \
   V(ERR_WASI_NOT_STARTED, Error)                                               \
   V(ERR_WORKER_INIT_FAILED, Error)                                             \
+  V(ERR_WORKER_MESSAGEPORT_CLOSED, Error)                                      \
```

Comment:
Actually, just noticing, we do have an error code for this already: `ERR_CLOSED_MESSAGE_PORT`

Reply:
Looks like that error was removed.

https://github.com/nodejs/node/commit/73370b4584

https://nodejs.org/dist/latest-v14.x/docs/api/errors.html#errors_err_closed_message_port

Reply:
We can add it back :)

Generally speaking, if we already have an error code, even if it’s no longer in use, I think we should use it.

(Using `ERR_WORKER_MESSAGEPORT_CLOSED` would also not be ideal because this isn’t inherently related to Workers.)

Reply:
Not glad to say this, but the error is back! Hahah